modules/gdscript/gdscript_parser.cpp

```diff
@@ -458,9 +458,9 @@ GDScriptParser::Node *GDScriptParser::_parse_expression(Node *p_parent, bool p_s
 			if (!validating) {
 
 				//this can be too slow for just validating code
-				if (for_completion && ScriptCodeCompletionCache::get_singleton()) {
+				if (for_completion && ScriptCodeCompletionCache::get_singleton() && FileAccess::exists(path)) {
 					res = ScriptCodeCompletionCache::get_singleton()->get_cached_resource(path);
```

Comment:
I still got some of those error logs that where caused when this was called, so I changed the condition above.
Is that ok, or will it cause other issues with completion of paths etc. ...?

Reply:
I'm not sure. I think I have yet to understand the renaming that's going on with resources.

Reply:
Hmm ... renaming as I understand it shouldn't matter here as for_completion being true should indicate, that it's called from an editor that uses the original files and not converted/renamed ones like the export ... but I might be wrong.
I also had a look at get_cached_resource, and it really seems to just be a cache for those resources (I was a little concerned there is more going on regarding completion as that method comes from that CompletionCache Singleton).